### DISCUSSION TITLE
Create and run Playwright test sets using tags and grep

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#338] [Announcements]
# Create and run Playwright test sets using tags and grep

At some point in time you'll want to run a subset of your tests. Organizing your tests with Playwright test tags helps you to do this in a clean way. Using tags allows you to target specific tests in specific scenarios, and can help to scale your pipeline(s).

https://timdeschryver.dev/blog/create-and-run-playwright-test-sets-using-tags-and-grep

**Comment by rajeshomprakash** [comment]
Hi Tim thank you for the wonderful tutorial. I have a query . Lets say i have three tests in a file
1> "test one"
2> "one test one"
3> "Three test one"

now i want to run exactly first test which is test one. 
i used --grep "test one$" this runs all the tests and i used "^test one$" . This is giving error no tests found
